content/docs/examples/userauthentication.md

```diff
@@ -78,47 +107,69 @@ We need one more value from Auth0: **the `issuer_url` for your Auth0 account**.
 {{<break 2>}}
 
 ### Creating our Authorizer
-We can now create an `Authorizer` for our Cdev API. 
+We can now create an `Authorizer` for our Cdev API. An `Authorizer` is a function that adds an additional layer of access control to an API endpoint. For example, preventing an authenticated users from accessing data that requires admin authorization.
+
+{{<tool_tip key="info" summary="More About Authorizers">}}
+For more information about Authorizers you can:
+
+Get an understanding of the differences between **[Authentication and Authorization](https://auth0.com/docs/get-started/identity-fundamentals/authentication-and-authorization)**.
+
+Learn about **[AWS Lambda Authorizors](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-use-lambda-authorizer.html)**.
```

Comment:
This is something slightly different then what we are providing. It is actually just the JWT authorizer mechanism, in the next link, that we are setting up